src/cases/components/roles-and-access/roles-and-access.component.ts

```diff
@@ -1,14 +1,20 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { CaseView } from '@hmcts/ccd-case-ui-toolkit';
-import { CaseRole } from '../../../../api/workAllocation2/interfaces/caseRole';
 import { LocationInfo } from 'src/app/store/reducers/app-config.reducer';
```

Comment:
not your change, but could you change the import from src to relative path

Reply:
changed now